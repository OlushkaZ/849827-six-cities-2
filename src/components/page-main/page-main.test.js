import React from 'react';
import renderer from 'react-test-renderer';
import PageMain from './page-main.jsx';

it(`PageMain correctly renders after relaunch`, () => {
  const tree = renderer
 .create(<PageMain
   offers = {[``, ``]}
   // minutes={0}
   // onClick={jest.fn()}
 />)
 .toJSON();
  expect(tree).toMatchSnapshot();
});
