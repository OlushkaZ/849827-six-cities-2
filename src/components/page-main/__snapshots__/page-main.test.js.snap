// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageMain correctly renders after relaunch 1`] = `null`;
